src/lib/x509/certstor_system/info.txt

```diff
@@ -0,0 +1,8 @@
+<defines>
+CERTSTOR_SYSTEM -> 20190411
+</defines>
+
+<os_features>
+apple_keychain
+filesystem
```

Comment:
It's either `apple_keychain` or `filesystem`, isn't it? If both are needed, `System_Certificate_Store` will only ever be included on macOS.

Reply:
Syntax here is confusing, each line represents an alternative. AND is done with ',', like in https://github.com/randombit/botan/blob/master/src/lib/rng/system_rng/info.txt#L5

Reply:
Out of curiosity: Is there a write-up of the build system somewhere? So far I always defaulted to reading `./configure.py`...

Reply:
There is not. But if I'm going to force people to deal with a one-off build system it really should be documented.

Reply:
@reneme Still needs work but https://github.com/randombit/botan/blob/master/doc/manual/configure.rst

Reply:
Thanks for the overview! :)